src/JavaScriptHighlighter/JavaScriptHighlighter.css

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
```

Comment:
Can you use the [Webpack `BannerPlugin`](https://webpack.js.org/plugins/banner-plugin/) for this?

Reply:
That would be much nicer. I was asked to have the banner in every _source_ file, though.